content/guides/building/get-source-git.md

```diff
@@ -253,7 +264,8 @@ each commit and each version of a commit so make sure to get the correct one
 from Gerrit:
 
 ```sh
-git fetch "ssh://user@git.haiku-os.org/haiku" refs/changes/28/3228/4 && git cherry-pick FETCH_HEAD
+git fetch "ssh://user@git.haiku-os.org/haiku" refs/changes/28/3228/4
+git cherry-pick FETCH_HEAD
```

Comment:
here the && is really important. Otherwise you will end up checking out something else (whatever the FETCH_HEAD was set to)

Reply:
Oops.